File: round1077_div2/pA.py
def solve():
    n = int(input())
    arr = []
    curr = (n + 1) // 2
    for i in range(1, n + 1):
        arr.append(curr)
        curr += i * (-1 if i % 2 == 0 else 1)
    print(" ".join([str(c) for c in arr]))

t = int(input())
for _ in range(t):
    solve()

File: round1088/pA.cpp
#include <vector>
#include <iostream>

using namespace std;

void solve() {
    int n;
    cin >> n;
    vector<int> arr(n);
    for (int i=0; i<n; i++) {
        cin >> arr[i];
    }

    if (n == 1) {
        cout << 1 << endl;
    }
    else {
        for (int i=0; i<n; i++) {
            cout << 2 << ' ';
        }
        cout << endl;
    }
}

int main() {
    int t;
    cin >> t;
    for (int i=0; i<t; i++) {
        solve();
    }
}